projects/observability/src/pages/saved-queries/saved-queries.service.ts

```diff
@@ -0,0 +1,104 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { PreferenceService, SubscriptionLifecycle, UserTraits } from '@hypertrace/common';
+import { Filter } from '@hypertrace/components';
+import { ScopeQueryParam } from '../explorer/explorer.types';
+
+const BASE_URL = 'https://hus.concierge.stage.razorpay.in/v1';
```

Comment:
@akashmane1598 Please give me a heads up when the `/user-preferences` endpoint starts working. Will change this.